src/textFeature.js

```diff
@@ -60,6 +60,12 @@ var feature = require('./feature');
  *      stroke color.  May include opacity.
  * @property {geo.geoColor|function} [style.textStrokeWidth=0] Text stroke
  *      width in pixels.
+ * @property {number|function} [style.renderedZone] If this is a positive
```

Comment:
@manthey i am not entire sure if the name of the variable truly reflects the intention. Would you be open to call it something like renderThreshold or viewportThreshold or something else?

Reply:
Of those, I think `renderThreshold` would be clearer, or even a longer combination like `renderViewportThreshold`.  Do you have a preference?

Reply:
renderThreshold seems good enough for me now.  Since we do not use the word viewport otherwise, I would have chosen renderViewportThreshold.

the code LGTM

Reply:
Updated to `renderThreshold` and rebased to current master.